src/clr-core/internal/utils/dom.ts

```diff
@@ -17,3 +17,23 @@ export function getElementWidthUnless(element: HTMLElement, unless: boolean) {
   }
   return '';
 }
+
+export type HTMLAttributeTuple = [string, string | boolean];
+
+export function setAttributes(element: HTMLElement, attributeTuples: HTMLAttributeTuple[]) {
+  attributeTuples.forEach(tup => {
+    const [attr, val] = tup;
+
+    if (val === false || val === null) {
+      element.removeAttribute(attr);
+    } else {
+      element.setAttribute(attr, val + '');
+    }
+  });
+}
+
+export function removeAttributes(element: HTMLElement, attrs: string[]) {
```

Comment:
...and we can remove attributes too

`removeAttributes(['title', 'style', 'aria-label'])`